.github/workflows/publish-dev.yml

```diff
@@ -34,15 +34,15 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
 
       - name: Install JS dependencies
-        run: yarn
+        run: npm install
         working-directory: bindings/wasm
 
       - name: Build WASM bindings
-        run: yarn build
+        run: npm run build
         working-directory: bindings/wasm
 
       - name: Publish WASM bindings to NPM
-        run: yarn publish --tag dev --access public --loglevel silly
+        run: npm publish --tag dev --access public
```

Comment:
let's keep an eye on this next time we publish

Reply:
Indeed, and there was a difference between which files were included by `npm` vs `yarn` which required a slight change to `package.json`:
```
  "files": [
    "web/*",
    "node/*"
  ],
```

Otherwise, I did test that the package contents were the same:

> Tested a dry-run of publishing the package to npm, verified package contents are equivalent between `yarn pack` and `npm pack` after changes.